src/labels/CustomLabels.labels

```diff
@@ -1609,6 +1609,14 @@
         <shortDescription>Error when user does not have a permission to change an RD originated in Elevate</shortDescription>
         <value>Recurring Donations from Elevate Payment Services can&apos;t be modified in NPSP except by the Elevate Payment Services Integration user.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateNotSupportErrorMessage</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when the functionality is not supported for Elevate Recurring Donation.</shortDescription>
+        <value>This functionality is currently not supported for Recurring Donations generated from Elevate Payment Services.</value>
```

Comment:
@andrewyu-salesforce , has the message been approved by @sam-knox ?
CC @smarovich

Reply:
This is the new message @sam-knox  provided on the WI. Not approve on the PR yet.

Reply:
It looks good to me! (I'm filling in for Sam). Thanks!